packages/itwinui-css/src/table/base.scss

```diff
@@ -142,8 +142,6 @@
 
 @mixin iui-table-column-manager {
   padding: var(--iui-size-xs);
-  max-block-size: 17.25rem; // Show approximately 8.5 checkboxes
-  overflow-y: auto;
```

Comment:
I'm trying to understand the issue, but not able to reproduce on Chrome or Firefox.

<img width="166" alt="Screenshot" src="https://github.com/user-attachments/assets/709a7f86-ac02-401e-8138-8aca903fe12d" />

<img width="166" alt="Screenshot" src="https://github.com/user-attachments/assets/155a59cc-9fd3-4499-8d5c-7d82156a2f8f" />